lib/plugins/aws/provider/awsProvider.js

```diff
@@ -116,6 +118,42 @@ class AwsProvider {
       AWS.config.httpOptions.agent = new HttpsProxyAgent(proxyOptions);
     }
 
+    const ca = process.env.ca
+      || process.env.HTTPS_CA
+      || process.env.https_ca;
+
+    const caCerts = [];
+
+    if (ca) {
+      // Can be a single certificate or multiple, comma separated.
+      const caArr = ca.split(',');
+      // Replace the newline -- https://stackoverflow.com/questions/30400341
+      caArr.forEach((cert) => {
+        caCerts.push(cert.replace(/\\n/g, '\n'));
+      });
+    }
+
+    const cafile = process.env.cafile
+      || process.env.HTTPS_CAFILE
+      || process.env.https_cafile;
+
+    if (cafile) {
+      // Can be a single certificate file path or multiple paths, comma separated.
+      const caPathArr = cafile.split(',');
```

Comment:
Same here

Reply:
Updated to use concat and map since the code change allows users to both specify a cert as a string and as a file simultaneously.